modules/database/src/interfaces/ICms.ts

```diff
@@ -0,0 +1,44 @@
+interface CrudOperation {
```

Comment:
Do we really need this extra type? You're using type inference in `ICms` anyway.